proposals/nnnn-configurable-time-before-shutdown.md

```diff
@@ -0,0 +1,78 @@
+# Configurable time before shutdown
+
+* Proposal: [SDL-NNNN](nnnn-configurable-time-before-shutdown.md)
+* Author: [Alexander Kutsan](https://github.com/LuxoftAKutsan)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core]
+
+## Introduction
+
+To prevent missing logs in SDL shutdown add Additional ini file options:
+ - Write all logs to file system before shutdown 
+ - Maximum time of sdl shutting down
+ 
+## Motivation
+
+In debug mode SDL produce a lot of logs in is asynchronous mode. 
```

Comment:
@LuxoftAKutsan
SDL produce**s** a **tone** of logs in **the** asynchronous mode **as well as in some other** use cases (like video streaming or big put file),